app/src/main/java/org/mozilla/fenix/onboarding/OnboardingRadioButton.kt

```diff
@@ -51,12 +45,13 @@ class OnboardingRadioButton(context: Context, attrs: AttributeSet) : AppCompatRa
 
     private fun updateRadioValue(isChecked: Boolean) {
         this.isChecked = isChecked
-        Settings.getInstance(context).preferences.edit().putBoolean(context.getString(key), isChecked)
-            .apply()
+        Settings.getInstance(context).preferences.edit {
```

Comment:
Where is this being applied or committed to prefs?

Reply:
This is an Android KTX function that calls `apply` after.